webapp/components/about_build_modal.jsx

```diff
@@ -167,7 +167,15 @@ export default class AboutBuildModal extends React.Component {
                             />
                         </div>
                     </div>
-                    <div className='about-modal__hash form-group padding-top x2'>
+                    <div className='about-modal__notice form-group padding-top x2'>
+                        <p>
+                            <FormattedHTMLMessage
+                                id='about.notice'
+                                defaultMessage='Mattermost is made possible by the open source software used in our <a href="https://about.mattermost.com/platform-notice-txt/" target="_blank">platform</a>, <a href="https://about.mattermost.com/desktop-notice-txt/" target="_blank">desktop</a> and <a href="https://about.mattermost.com/mobile-notice-txt/" target="_blank">mobile apps</a>.'
```

Comment:
Small thing. I don't think "apps" is supposed to be part of the "mobile" link

Reply:
@hmhealey actually, the rough design and the description did not match in Jira ticket. I could easily change it but honestly, I think apps should be included on link.

Reply:
@hmhealey updated the link